app/src/main/java/com/insudev/euvictodo/newTodoFragment/mvi/NewTodoPresenter.kt

```diff
@@ -1,23 +1,22 @@
 package com.insudev.euvictodo.newTodoFragment.mvi
 
 import com.hannesdorfmann.mosby3.mvi.MviBasePresenter
-import com.insudev.euvictodo.MainList.TodoRetroService
 
 
 class NewTodoPresenter : MviBasePresenter<NewTodoView, NewTodoState>() {
-    private val retroService = TodoRetroService()
+
     private val reducer: NewTodoReducer = NewTodoReducer()
     override fun bindIntents() {
 
-        var newTodoContent = intent { it.newTodoContent }
+        val newTodoContent = intent { it.newTodoContent }
             .map { ContentChangeResult.Completed(it) as ContentChangeResult }
```

Comment:
tutaj masz kolejny wrapper na rezultaty. Powinien to być jeden generyczny. Obsługa błędu powinna być w jakimś serwisie